lib/iris/tests/test_netcdf.py

```diff
@@ -1,4 +1,4 @@
-# (C) British Crown Copyright 2010 - 2018, Met Office
+# (C) British Crown Copyright 2010 - 2019, Met Office
```

Comment:
After some discussion with @lbdreyer , I think we're agreed that we'd rather not add more to this sourcefile.
The reason is that the files `lib/iris/tests/test*.py` are all basically ancient legacy code, and we'd rather have all testing either under `lib/iris/tests/integration`, or in appropriate sub-directories of `lib/iris/tests/unit`.
( ultimately it would be nice to tidy up the entire structure, but we never seem to find the time! )

In this case, I think you could usefully put this code in `lib/iris/tests/integration/test_netcdf.py` instead.
Is that ok ?   I couldn't see any technical difficulty with it.